.changeset/heavy-pots-give.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/memory': patch
+---
+
+Added persistent reminder conversations and asynchronous `ask_memory` questions with correlated partial and terminal replies.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)